scipy/stats/_distribution_infrastructure.py

```diff
@@ -0,0 +1,5158 @@
+
+        """
+        return self.moment(2, kind='central', method=method)
+
+    def standard_deviation(self, *, method=None):
+        r"""Standard deviation (square root of the second central moment)
+
+        Parameters
+        ----------
+        method : {None, 'formula', 'transform', 'normalize', 'quadrature', 'cache'}
+            Method used to calculate the central second moment. Not
+            all methods are available for all distributions. See
+            `moment` for details.
+
+        See Also
+        --------
+        variance
+        mean
+        moment
+
+        Examples
+        --------
+        Instantiate a distribution with the desired parameters:
+
+        >>> from scipy import stats
+        >>> X = stats.Normal(mu=1., sigma=2.)
+
+        Evaluate the standard deviation:
+
+        >>> X.standard_deviation()
+        2.0
+        >>> X.standard_deviation() == X.moment(order=2, kind='central')**0.5 == X.sigma
+        True
+
+        """
+        return np.sqrt(self.variance(method=method))
+
+    def skewness(self, *, method=None):
+        r"""Skewness (standardized third moment)
+
+        Parameters
+        ----------
+        method : {None, 'formula', 'general', 'transform', 'normalize', 'cache'}
+            Method used to calculate the standardized third moment. Not
+            all methods are available for all distributions. See
+            `moment` for details.
+
+        See Also
+        --------
+        moment
+        mean
+        variance
+
+        Examples
+        --------
+        Instantiate a distribution with the desired parameters:
+
+        >>> from scipy import stats
+        >>> X = stats.Normal(mu=1., sigma=2.)
+
+        Evaluate the skewness:
+
+        >>> X.skewness()
+        0.0
+        >>> X.skewness() == X.moment(order=3, kind='standardized')
+        True
+
+        """
+        return self.moment(3, kind='standardized', method=method)
+
+    def kurtosis(self, *, method=None, convention='non-excess'):
+        r"""Kurtosis (standardized fourth moment)
+
+        By default, this is the standardized fourth moment, also known as the
+        "non-excess" or "Pearson" kurtosis (e.g. the kurtosis of the normal
+        distribution is 3). The "excess" or "Fisher" kurtosis (the standardized
+        fourth moment minus 3) is available via the `convention` parameter.
+
+        Parameters
+        ----------
+        method : {None, 'formula', 'general', 'transform', 'normalize', 'cache'}
+            Method used to calculate the standardized fourth moment. Not
+            all methods are available for all distributions. See
+            `moment` for details.
+        convention : {'non-excess', 'excess'}
+            Two distinction conventions are available:
+
+            - ``'non-excess'``: the standardized fourth moment; the "Pearson" kurtosis.
+            - ``'excess'``: the standardized fourth moment minus 3; the "Fisher"
+                            kurtosis.
+
```

Comment:
I think the implied pause of the semicolons here makes it read like we're listing off pairs of separate things. How about
```
   - ``'non-excess'``: the standardized fourth moment (Pearson's kurtosis).
   - ``'excess'``: the standardized fourth moment minus 3 (Fisher's kurtosis).
```
?

Reply:
OK.